public/css/style.css

```diff
@@ -47,3 +47,31 @@ a.navbar-brand {
 select#languages {
     height: 30em;
 }
+
+h2 {
+	font-size: 1.3em;
+	font-weight: normal;
+	color: #000000;
+}
+h3 {
+	font-size: 1.2em;
+	font-weight: normal;
+	color: #000000;
+}
+
+h2.help {
```

Comment:
This looks out of place. Stick with bootstrap styles.

Reply:
OK. I do like those lines under the headings, though, as they don't have the go-up buttons look like they are hanging in the air. :)